Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/AbilData.xml

```diff
@@ -11740,4 +11779,161 @@
         <EditorCategories value="Race:Neutral,AbilityorEffectType:Structures"/>
         <QueueSize value="7"/>
     </CAbilQueue>
+    <CAbilEffectInstant id="AP_ZergMercCompoundBroodWar">
+        <Effect index="0" value="AP_ZergMercCompoundBroodWarTabSet"/>
+        <CmdButtonArray index="Execute" DefaultButtonFace="AP_ZergMercBroodWar" Requirements="AP_ZergMercCompoundCanSwitchBroodWar"/>
+        <Cost index="0"/>
+    </CAbilEffectInstant>
+    <CAbilEffectInstant id="AP_ZergMercCompoundSC2">
+        <Effect index="0" value="AP_ZergMercCompoundSC2Set"/>

```

Comment:
`AP_ZergMercCompoundSC2TabSet` (otherwise it won't work)

Reply:
yeah, missed that since there aren't any actual sc2 mercs yet.